.devcontainer/configuration.yaml

```diff
@@ -6,3 +6,4 @@ logger:
     pyze: debug
     custom_components.renault: debug
 
+ptvsd:
```

Comment:
Can you open a separate PR for this, and explain why this is needed?

Reply:
Deactivated with a comment . Is it OK ?